src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,47 +1,133 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 11/03/17
  */
+import com.cs56fitnessapp.models.workout.Workout;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Date;
+
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** date */
+    private LocalDate date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
+    /** WorkoutList */
+    private ArrayList < Workout > workoutList;
+
+    /** MealList */
+    private ArrayList < Meal > mealList;
+
+    public Day (LocalDate date, User user){
+      this.date = date;
+      this.user = user;
+      this.workoutList = new ArrayList <Workout>();
+      this.mealList = new ArrayList <Meal>();
+    }
+
+    public boolean addWorkout(Workout workout){
+      this.workoutList.add(workout);
+      return true;
+    }
+
```

Comment:
`add` method in `ArrayList` API returns `true`, so this one should be implemented the following way:

```java
public boolean addWorkout(Workout workout) {
    return this.workoutList.add(workout);
}
```